src/templates/vibso_terms.tsv

```diff
@@ -14,3 +14,4 @@ obo:VIBSO_0000010	Raman spectrometer configuration	A device setting that represe
 obo:VIBSO_0000011	excitation wavelength	A physical quality of the light source that is used used to excite a system to a higher energy state.	DOI:10.1351/goldbook	owl:Class	wavelength					ORCID:0000-0002-1595-3213			This class needs to be defined in PATO or another more general ontology for such qualities.
 obo:VIBSO_0000012	diffraction grating	A device that is an optical component with a periodic structure that diffracts light into several beams travelling in different directions (i.e. different diffraction angles)	https://en.wikipedia.org/wiki/Diffraction_grating	owl:Class	device					ORCID:0000-0002-1595-3213			This class should better be defined in a more general device ontology or a domain ontology with the scope of optical devices.
 obo:VIBSO_0000013	excitation wavelength setting	A setting datum that specifies the wavelength of an electromagnetic radiation source (e.g. a laser) which has the function to excite the energy state of a target material.		owl:Class	setting datum					ORCID:0000-0002-1595-3213			This class should better be defined in a more general device ontology or a domain ontology with the scope of optical devices.
+obo:VIBSO_0000014	objective magnification factor setting	A setting datum that specifies the magnification factor of the objective collecting the scattered light.		owl:Class	setting datum					ORCID:0000-0002-1595-3213		:experimentXY obo:VIBSO_0000014 '100x' .	
```

Comment:
```suggestion
obo:VIBSO_0000014	objective magnification factor setting	A setting datum that specifies the magnification factor of the objective collecting the scattered light.		owl:Class	setting datum					ORCID:0000-0001-7694-5519; ORCID:0000-0002-2239-3955		:experimentXY obo:VIBSO_0000014 '100x' .	
```

Reply:
done in [4562d5a](https://github.com/NFDI4Chem/VibrationalSpectroscopyOntology/pull/25/commits/4562d5ad60888e9f7382f9c8366b4d54e65021e9)